src/index.css

```diff
@@ -0,0 +1,69 @@
+.floating-label-field {
+    border-width: 0;
+    color: rgba(#2c3e50, .75);
```

Comment:
We need to avoid having fixed colors and use css variables instead, as we will need to implement light/dark mode feature. Having fixed colors will make this very difficult to implement.